admin/src/main/resources/application-mysql.properties

```diff
@@ -1,7 +1,7 @@
 spring.datasource.driver-class-name=com.mysql.jdbc.Driver
 spring.datasource.url=jdbc:mysql://localhost:3306/weplantaforest
-spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.username=weplantaforest
+spring.datasource.password=weplantaforest
 spring.jpa.database=MYSQL
```

Comment:
please create own(git ignored) property files, if you have other mysql properties
now i have to do this or i have to change these properties back to root/root to get the mysql connector working